examples/controls/app.js

```diff
@@ -1,13 +1,14 @@
 /* global window */
 import React, {Component} from 'react';
 import {render} from 'react-dom';
-import MapGL, {Marker} from 'react-map-gl';
+import MapGL, {Marker, Popup} from 'react-map-gl';
 
 import CityPin from './components/city-pin';
 
 import CITIES from './data/cities.json';
 
 import './app.css';
+import 'mapbox-gl/dist/mapbox-gl.css';
```

Comment:
I also added style-loader and css-loader to the webpack config.
Browserify has a css transform as well.